HIP/hip-1.md

```diff
@@ -74,11 +75,15 @@ Once the champion has discovered with the Hedera community the acceptability of
 
 The standard HIP workflow is:
 
-* You, the HIP author, fork the HIP repository, and create a file named hip-0000-my-feature.md (where “my-feature” is descriptive) that contains your new HIP. Use 0000 as your draft HIP number. This file should be created from the HIP template.
+* You, the HIP author, fork the HIP repository, and create a file named `hip-0000-my-feature.md` (where "my-feature" is descriptive) that contains your new HIP. Use 0000 as your draft HIP number. This file should be created from the HIP template.
+
+* In the "Type" header field, enter "Standards Track", "Informational", or "Process" as appropriate, and for "Status" field enter "Draft".
 
-* In the “Type” header field, enter “Standards Track”, “Informational”, or “Process” as appropriate, and for “Status” field enter “Draft”.
+* Push this to your Github fork and submit a draft pull request ([https://github.blog/2019-02-14-introducing-draft-pull-requests/](https://github.blog/2019-02-14-introducing-draft-pull-requests/)).
 
-* Push this to your Github fork and submit a pull request.
+* As you iterate through the community feedback, you add changes to the HIP which will be reflected in the draft pull request.
+
+* Once you finalize the changes related to the feedback, convert the pull request from "Draft" to "Review". This will signal an editor to review your HIP.
```

Comment:
We should add note about the DCO requirements before it is submitted for review.
Add some reference links: https://www.secondstate.io/articles/dco/, https://github.com/apps/dco

Reply:
Noted, updated.